Airports/EGPA/Positions.txt

```diff
@@ -1,4 +1,4 @@
-EGPA_APP:Kirkwall Approach:118.300:PAA:S:EGPA:APP:-:-:::N058.57.29.000:W002.54.18.000::
-EGPA_TWR:Kirkwall Tower:118.300:PAT:S:EGPA:TWR:-:-:::N058.57.29.000:W002.54.18.000::
-EGPA_I_TWR:Kirkwall Information:118.300:PAI:S:EGPA_I:TWR:-:-:::N058.57.29.000:W002.54.18.000::
+EGPA_APP:Kirkwall Approach:118.300:PAA:S:EGPA:APP:-:-:::N058.57.29.000:W002.54.02.000::
+EGPA_TWR:Kirkwall Tower:118.300:PAT:S:EGPA:TWR:-:-:::N058.57.29.000:W002.54.02.000::
+EGPA_I_TWR:Kirkwall Information:118.300:PAI:S:EGPA_I:TWR:-:-:::N058.57.29.000:W002.54.02.000::
 EGPA_ATIS:Kirkwall ATIS:108.600:PAI:S:EGPA:ATIS:-:-::::::
```

Comment:
Duplicate position identifier. I suggest we just use PAT, since only one of them will ever be on at the same time. Or perhaps we should use PAIT?

```suggestion
EGPA_I_TWR:Kirkwall Information:118.300:PAT:S:EGPA_I:TWR:-:-:::N058.57.29.000:W002.54.02.000::
EGPA_ATIS:Kirkwall Information:108.600:PAI:S:EGPA:ATIS:-:-::::::
```